docs_app/.firebaserc

```diff
@@ -0,0 +1,11 @@
+{
+  "targets": {
+    "rxjs-dev": {
+      "hosting": {
+        "stable": [
+          "rxjs-dev"
```

Comment:
I don't actually know much about this configuration. Can you explain what this is doing?

Reply:
sure, so we have one firebase project called rxjs-dev, we only use the hosting feature of firebase. Within firebase though we host now one "site" per version and those get an alias. So if you do `firebase deploy --only hosting:stable` it will deploy the current dist folder in the stable site. It's kinda like an alias for the site that is used within this branch